crates/runtime/src/dataaccelerator/spice_sys/kafka/duckdb.rs

```diff
@@ -0,0 +1,204 @@
+/*
+Copyright 2024-2025 The Spice.ai OSS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+use super::{KAFKA_TABLE_NAME, KafkaMetadata, KafkaSys, Result};
+use datafusion_table_providers::sql::db_connection_pool::duckdbpool::DuckDbConnectionPool;
+use std::sync::Arc;
+
+impl KafkaSys {
```

Comment:
Question (somewhat related) about implementing per-dataaccelerator SQL functions. Could we use the Datafusion SQL API to have a single `CREATE TABLE IF NOT EXISTS....` function that could then use the `sqlparser`s underneath to convert to each underlying data accelerator SQL dialect (i.e. avoid needing `.../spice_sys/kafka/duckdb.rs`, `.../spice_sys/kafka/postgres.rs`, etc)?

Reply:
DataFusion doesn't support full DDL statements (including CREATE TABLE), so it wouldn't be representable in a LogicalPlan and thus unable to be unparsed into a different dialect.